### PR TITLE
Completed CSS Canvas Medical - Joshua Baker

### PR DESCRIPTION
**Pull request by JoshuaAaronBaker**
Screen Shot 2022-07-07 at 1 20 03 AM
Screen Shot 2022-07-07 at 1 20 28 AM
Screen Shot 2022-07-07 at 1 20 38 AM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Josh! A few things:
1. You are missing the overlay over the hero image, it is too bright as compared to the example.
2. The text color of the "Ready to learn more?" headline does not match the example
3. The links in the final row are misaligned with the rest of the content

Please revise and submit a new screenshot with your revision.

**Comment by JoshuaAaronBaker** [comment]
Screen Shot 2022-07-10 at 11 56 22 PM
Screen Shot 2022-07-10 at 11 56 33 PM
Screen Shot 2022-07-10 at 11 56 42 PM


I dont understand why the Ready to learn more text is a different font color.
Screen Shot 2022-07-10 at 11 59 55 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Have you tried inspecting the element to see what styling is currently being applied to it? It may be a specificity issue.

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.container2 {
+  max-width: 1000px;
+  margin: auto;
+}
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:13]: You shouldn't make numbered classes like this. You could make this a utility class named something like `large-container` that only has the `max-width` property declaration, and apply it in addition to the actual `container` class

File: css-canvas-medical/styles.css [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+  background-color: rgb(250 250 250);
+}
+
+.dark-background {
+  background-color: rgb(82 83 81);
+  padding-top: 5%;
+  padding-bottom: 5%;
+}
+
+.margin3 {
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:41]: What does the `3` represent?

File: css-canvas-medical/styles.css [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+.overlay {
+  position: absolute;
+  left: 5%;
+  right: 5%;
+  top: 5%;
+  width: 100%;
+  height: 100%;
+}
+
+.padding1 {
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:120]: `1`?

File: css-canvas-medical/styles.css [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+  font-style: normal;
+  font-weight: 500;
+  color: rgb(110 110 110);
+}
+
+.space-evenly {
+  justify-content: space-evenly;
+}
+
+.padding {
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:160]: How much padding?

File: css-canvas-medical/styles.css [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+
+.p2-font {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: rgb(110 110 110);
+}
+
+.sec-3-font {
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:188]: What is `sec-3`?

File: css-canvas-medical/styles.css [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+}
+
+.h2font {
+  font-family: "Tiempos Headline", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  color: rgb(110 110 110);
+}
+
+.p2-font {
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:180]: `p2`?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**